control/available_plugin.go

```diff
@@ -285,22 +285,15 @@ func (a *availablePlugin) healthCheckFailed() {
 		"aplugin": a,
 	}).Warning("heartbeat missed")
 	a.failedHealthChecks++
-	if a.failedHealthChecks >= DefaultHealthCheckFailureLimit {
-		log.WithFields(log.Fields{
-			"_module": "control-aplugin",
-			"block":   "check-health",
-			"aplugin": a,
-		}).Warning("heartbeat failed")
-		pde := &control_event.DeadAvailablePluginEvent{
-			Name:    a.name,
-			Version: a.version,
-			Type:    int(a.pluginType),
-			Key:     a.key,
-			Id:      a.ID(),
-			String:  a.String(),
-		}
-		defer a.emitter.Emit(pde)
+	pde := &control_event.DeadAvailablePluginEvent{
+		Name:    a.name,
+		Version: a.version,
+		Type:    int(a.pluginType),
+		Key:     a.key,
+		Id:      a.ID(),
+		String:  a.String(),
 	}
+	defer a.emitter.Emit(pde)
```

Comment:
It still seems weird to emit a DeadAvailablePluginEvent just for missing 1 heartbeat. What is the reasoning behind removing the failedChecks >= healthcheckfailurelimit?